dask/base.py

```diff
@@ -18,9 +17,47 @@
 __all__ = ("Base", "compute", "normalize_token", "tokenize", "visualize")
 
 
-class Base(object):
+class DaskIntf(object):
```

Comment:
I think this is fine, though I would call it `DaskInterface` instead of abbreviating `Intf`. It took me a while to figure that one out.